markdown/en/ff-suggest.md

```diff
@@ -62,20 +69,23 @@ Repeat this process for all configured suggest types.
 </ff-suggest>
 ```
 
-Note the triple curly braces in `<span>{{{name}}}</span>`.
-These are required because `ff-suggest` inserts HTML here.
-The matched string from your search box is wrapped in a
-`<span class="query">` tag to allow you to highlight the matched text
-through CSS. If you were to use the regular double curly braces and type
-for example "back", the result would be
-`<span class="query">Back</span>packs` instead of "**Back**packs".
+**Note** the triple curly braces in `<span>{{{name}}}</span>`, which
+enable the rendering of HTML code inserted via Polymer's data binding.
+`ff-suggest` inserts HTML here and the matched string from your search
+box is wrapped in a `<span class="query">` tag to allow you to highlight
+the matched text through CSS. If you used the regular double curly
+braces and typed "back" in the search box, the HTML source code would be
+shown as a plain text: `<span class="query">Back</span>packs`
+rather than the rendered: "**Back**packs".
 
```

Comment:
Beside the nice highlighting of **Note** I actual liked the whole previous paragraph more.